runtime/jnichk/jnicheck.c

```diff
@@ -132,6 +133,11 @@ J9VMDllMain(J9JavaVM* vm, IDATA stage, void* reserved)
 				return J9VMDLLMAIN_FAILED;
 			}
 
+			if ((*hook)->J9HookRegisterWithCallSite(hook, J9HOOK_VM_CLASSES_REDEFINED, classesRedefinedHook, OMR_GET_CALLSITE(), NULL)) {
+				j9tty_err_printf(PORTLIB, "<JNI check utility: unable to hook event>\n");
```

Comment:
This could be more specific and name the hook.

Reply:
The other two don't, and these are essentially impossible situations.